actionview/lib/action_view/test_case.rb

```diff
@@ -71,11 +71,11 @@ def determine_default_helper_class(name)
         def helper_method(*methods)
           # Almost a duplicate from ActionController::Helpers
           methods.flatten.each do |method|
-            _helpers.module_eval <<-end_eval
+            _helpers.module_eval <<-END_EVAL, __FILE__, __LINE__ + 1
```

Comment:
We have instances of `<<-end_foo`, and instances of `<<-FOO`. I see no reason to introduce a third variant.